sync/rpc/server.go

```diff
@@ -180,7 +182,96 @@ func (s *SyncAgentServer) ReceiverLaunch(ctx context.Context, req *ReceiverLaunc
 	return &ReceiverLaunchReply{Port: int32(port)}, nil
 }
 
-func (*SyncAgentServer) BackupCreate(ctx context.Context, req *BackupCreateRequest) (*BackupCreateReply, error) {
+func RequiredMissingError(name string) error {
+	return fmt.Errorf("cannot find valid required parameter: %v", name)
+}
+
+func openBackingFile(file string) (*replica.BackingFile, error) {
+	if file == "" {
+		return nil, nil
+	}
+	return nil, fmt.Errorf("backing file is not supported")
+}
+
+func doCreateBackup(req *BackupCreateRequest) (string, *replica.Backup, error) {
```

Comment:
Don't copy the code. Refactor the code in the `/backup/` and make a function call there.